### PR TITLE
🌱 Update Kustomize deprecated syntax

### PR DESCRIPTION
**Pull request by peppi-lotta**
Update Kustomize deprecated syntax:
    
    - patchesStrategicMerge -> patches
    - patchesJson6902 -> patches
    - vars and varReference -> replacements
    - bases -> resources
    
    and bump version

**What this PR does / why we need it**:

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

### CODE REVIEW

**Comment by kashifest** [comment]
cc @lentzi90 @Rozzii @mboukhalfa @adilGhaffarDev @smoshiur1237 @kashifest

File: config/certmanager/certificate.yaml [hunk 1]
```diff
@@ -14,11 +14,11 @@ metadata:
...
   dnsNames:
-  - $(SERVICE_NAME).$(SERVICE_NAMESPACE).svc
-  - $(SERVICE_NAME).$(SERVICE_NAMESPACE).svc.cluster.local
+  - SERVICE_NAME.SERVICE_NAMESPACE.svc
+  - SERVICE_NAME.SERVICE_NAMESPACE.svc.cluster.local
   issuerRef:
     kind: Issuer
     name: selfsigned-issuer
-  secretName: $(SERVICE_NAME)-cert # this secret will not be prefixed, since it's not managed by kustomize
+  secretName: SERVICE_NAME-cert # this secret will not be prefixed, since it's not managed by kustomize
```
Comment (kashifest) [inline config/certmanager/certificate.yaml:24]: I think this needs now proper name like ipam-SOMETHING-service.cert

**Review by lentzi90 — COMMENTED** [review]
/approve

**Review by Rozzii — APPROVED** [review]
/lgtm

**Pull request merged**